argus/action_manager/windows.py

```diff
@@ -283,6 +286,60 @@ def is_dir(self, path):
         """
         return self._exists(path, self.PATH_CONTAINER)
 
+    def _new_item(self, path, item_type):
+        """Create a New-Item.
+
+        :param path:
+            Instance path of the new item.
+        :param item_type:
+            It can be `Directory` or `File`
+        """
+        cmd = "New-Item -Path '{}' -Type {} -Force".format(path, item_type)
+        self._client.run_command_with_retry(cmd=cmd,
+                                            command_type=util.POWERSHELL)
+
+    def mkdir(self, path):
+        """Create a directory in the instance if the path is valid.
+
+        :param path:
+            Remote path where the new file should be created.
+        """
+        if self.exists(path):
+            exceptions.ArgusCLIError("Directory already exists!")
+        else:
+            self._new_item(path, self._DIRECTORY)
+
+    def mkfile(self, path):
+        """Create a file in the instance if the path is valid.
+
+        :param path:
+            Remote path where the new directory should be created.
+        """
+        if self.is_file(path):
+            self._client.run_command_with_retry(
+                "echo $null >> '{}'".format(path),
+                command_type=util.POWERSHELL)
+        elif self.is_dir(path):
+            exceptions.ArgusCLIError("Path leads to a directory")
+        else:
+            self._new_item(path, self._FILE)
+
+    def touch(self, path):
+        """Update the access and modification time to the current time.
+
+        If the file doesn't exist, an empty file will be created
+        as side effect.
+        """
+        if self.is_dir(path):
```

Comment:
By echo-ing `$null` in the file the `LastWriteTime` and `LastAccessTime` members are changed, I used this approach  because `mkfile` should also "touch" the file.The special case is a directory where I need to set the dates by hand.